js-advanced-unit-5/vending-machines/money-functions.js

```diff
@@ -0,0 +1,50 @@
+const formatCurrency = amount => {
+  if (isNaN(amount)) {
+    return "Not a Number"
+  }
+
+  const amountRoundedAndFixedToTwo = (
+    Math.round(Math.abs(amount) * 100) / 100
+  ).toFixed(2)
+  const currency =
+    amount >= 0
+      ? "$" + amountRoundedAndFixedToTwo
+      : "-$" + amountRoundedAndFixedToTwo
```

Comment:
And maybe a one-line ternary with parens for readability
```suggestion
  return (amount >= 0) ? ("$" + formattedAmount) : ("-$" + formattedAmount)
```